client/src/components/AuditAdmin/Progress/JurisdictionDetail.tsx

```diff
@@ -274,34 +319,49 @@ const RoundStatusSection = ({
     }
 
     if (sampleCount.ballots === 0) return <p>No ballots sampled</p>
-    if (jurisdictionStatus === JurisdictionRoundStatus.COMPLETE)
-      if (auditSettings.auditType === 'BATCH_COMPARISON')
+    if (jurisdictionStatus === JurisdictionRoundStatus.COMPLETE) {
+      if (auditSettings.auditType === 'BATCH_COMPARISON') {
         return (
           <div>
             <p>Results finalized</p>
             <AsyncButton
-              onClick={async () => {
-                if (
-                  await unfinalizeBatchResults(
-                    electionId,
-                    jurisdiction.id,
-                    round.id
-                  )
-                )
-                  // Hack: for now just reload the whole page here instead of
-                  // properly refreshing the state from the server, since the
-                  // state is way high up in the component hierarchy
-                  window.location.reload()
-              }}
+              onClick={() =>
+                unfinalizeBatchResults({
+                  electionId,
+                  jurisdictionId: jurisdiction.id,
+                  roundId: round.id,
+                })
+              }
               intent="danger"
             >
               Unfinalize Results
             </AsyncButton>
           </div>
         )
-      else return <p>Data entry complete</p>
-    if (auditBoards.length === 0)
+      }
+      return (
+        <div>
+          <p>Data entry complete</p>
+          {auditSettings.online && (
```

Comment:
When audit boards are offline, an AA can't take any meaningful actions on audit boards, so I opted not to display the audit boards table in this case

This differs from support users, where we still display offline audit boards, just with no reopen option. Support users have other abilities, like clearing all audit boards, so displaying even offline audit boards to them makes more sense